libnavui-voice/src/main/java/com/mapbox/navigation/ui/voice/api/VoiceInstructionsTextPlayer.kt

```diff
@@ -17,19 +17,26 @@ import java.util.Locale
  */
 internal class VoiceInstructionsTextPlayer(
     private val context: Context,
-    private val language: String,
+    private var language: String,
     private val playerAttributes: VoiceInstructionsPlayerAttributes,
 ) : VoiceInstructionsPlayer {
 
     @VisibleForTesting
     internal var isLanguageSupported: Boolean = false
 
+    private var textToSpeechInitStatus: Int = TextToSpeech.ERROR
```

Comment:
```suggestion
    private var textToSpeechInitStatus: Int? = null
```

nitpick: could reserve the error state. status == TextToSpeech.SUCCESS works for both